eventmesh-registry-plugin/eventmesh-registry-etcd/src/main/java/org/apache/eventmesh/registry/etcd/service/EtcdCustomService.java

```diff
@@ -68,6 +68,9 @@ public List<EventMeshServicePubTopicInfo> findEventMeshServicePubTopicInfos() th
                 return eventMeshServicePubTopicInfoList;
             }
         } catch (Exception e) {
+            if(e instanceof InterruptedException) {
```

Comment:
I think it would be better to put it after the log message.

Reply:
ohk sure